lib/manageiq/messaging/kafka/client.rb

```diff
@@ -93,8 +93,10 @@ def rdkafka_connection_opts(options)
           result[:"sasl.password"]     = options[:password] if options[:password]
           result[:"ssl.ca.location"]   = options[:ca_file] if options[:ca_file]
           result[:"security.protocol"] = !!options[:ssl] ? "SASL_SSL" : "PLAINTEXT"
+          result[:"ssl.keystore.location"] = options[:keystore_location] if options[:keystore_location]
+          result[:"ssl.keystore.password"] = options[:keystore_password] if options[:keystore_password]
```

Comment:
Really minor but can we group these two with the `ssl.ca.location` option above?  Maybe actually just move the `security.protocol` up a line since that would keep things alphabetical